src/ApiServices/ApiServices.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+const signUpHandler = async ({ email, firstName, lastName, password }) => {
+  try {
+    const res = await axios.post(`/api/auth/signup`, {
+      email: email,
+      firstName: firstName,
+      lastName: lastName,
+      password: password,
```

Comment:
If the parameters received from the function and object keys have the same name then you can simply write it as 
```
const res=await axios.post("/api/auth/signup",{email,password,firstName,lastName})

```
It will also work in the same manner as the one you have written earlier

Reply:
Noted, will try next time